packages/dashboard/src/components/sidePanel/sections/index.css

```diff
@@ -19,6 +19,11 @@
   border-radius: 0.57rem;
 }
 
+/* remove default popover padding */
+.threshold-content-color-picker div.awsui_body_xjuzf_10pru_325.awsui_body-overflow-visible_xjuzf_10pru_339 {
```

Comment:
is this a reliable selector? I used data attributes to do this in the text widget commit.

Reply:
Replaced react-color lib with native color picker <input type="color">. This has been removed.